planet/clients/orders.py

```diff
@@ -271,18 +274,69 @@ async def download_asset(self,
                              progress_bar=progress_bar)
         return dl_path
 
+    @staticmethod
+    def __validate_checksum(manifest_data: dict,
```

Comment:
private - or, more specifically, non-public - functions start with just one underscore, so this would be `_validate_checksum()` ([ref](https://docs.python.org/3.7/tutorial/classes.html#private-variables))

```suggestion
    def _validate_checksum(manifest_data: dict,
```